### ISSUE TITLE
Printing without terminating new line causes VS debugger to hang

### ISSUE THREAD

**Issue by dslijepcevic** [Microsoft/VSLinux#134]
Stepping over `fprintf()` line in debugger (F10) does not work unless there is a terminating `\n` at the end of the string as `gdb` response gets intermixed with it:

Code:
```c
#include <stdio.h>

int main()
{
    fprintf(stderr, "can't step over this");
    return 0;
}
```

Output window:
```
Breakpoint 1, main () at buggy.c:5
5	    fprintf(stderr, "can't step over this");
can't step over this*stopped,reason="end-stepping-range",frame={addr="0x0000000000400588",func="main",args=[],file="buggy.c",fullname="/work/buggy.c",line="6"},thread-id="1",stopped-threads="all",core="0"
```

VS debugger probably misses this response due to a parse error and hangs waiting for another one.

Host:
Windows 10 Pro 1703
Visual Studio Community 2017 Version 15.0.26228.4 D15RTWSVC
Visual C++ for Linux Development 1.0.6

Target:
Linux ubuntu 4.4.0-72-generic #93-Ubuntu SMP Fri Mar 31 14:07:41 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
gcc version 5.4.1 20160904 (Ubuntu 5.4.1-2ubuntu1\~16.04)
GNU gdb (Ubuntu 7.11.1-0ubuntu1\~16.04) 7.11.1

**Comment by robotdad** [comment]
@paulmaybee this repros in 15.2 and 15.3 in gdb mode, gdbserver mode seems to work.

**Comment by paulmaybee** [comment]
This bug should be fixed by the change to support the Linux Console Window when running in gdb debugging mode.

**Comment by lukka** [comment]
fixed in #158 , the fix will be avail in VS2017 Update 5
